src/main/java/org/broadinstitute/hellbender/utils/mcmc/ParameterReader.java

```diff
@@ -1,5 +1,6 @@
 package org.broadinstitute.hellbender.utils.mcmc;
 
+import java.nio.file.Path;
```

Comment:
OK to delete this and ParameterWriter, I believe they are dead code.

Reply:
The compiler agrees with your assessment. Deleted!